packages/docusaurus-theme-common/src/utils/authorsUtils.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {translate} from '@docusaurus/Translate';
+import type {AuthorsListItem} from '@docusaurus/utils';
+
+export const translateAuthorsPageTitle = (): string =>
+  translate({
+    id: 'theme.authors.authorsPageTitle',
+    message: 'Authors',
+    description: 'The title of the author list page',
+  });
+
+export type AuthorLetterEntry = {
+  letter: string;
+  authors: AuthorsListItem[];
+};
+
+function getAuthorLetter(author: string): string {
+  return author[0]!.toUpperCase();
```

Comment:
What about authors without a name?

Reply:
For now authors without names are named internaly with their key https://github.com/facebook/docusaurus/blob/0a353236211ab6bd24e77a200af20a9c7d2eccb6/packages/docusaurus-plugin-content-blog/src/authors.ts#L244

So the route and the list of authors by letters are done with the key